<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tome 0.2.0</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuRecent_Projects">
     <property name="title">
      <string>Recent Projects</string>
     </property>
    </widget>
    <addaction name="actionNew_Project"/>
    <addaction name="separator"/>
    <addaction name="actionOpen_Project"/>
    <addaction name="actionSave_Project"/>
    <addaction name="separator"/>
    <addaction name="menuRecent_Projects"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="separator"/>
    <addaction name="actionManual"/>
    <addaction name="separator"/>
    <addaction name="actionReport_a_Bug"/>
    <addaction name="separator"/>
    <addaction name="actionReleases"/>
   </widget>
   <widget class="QMenu" name="menuFields">
    <property name="title">
     <string>Fields</string>
    </property>
    <addaction name="actionManage_Components"/>
    <addaction name="actionField_Definions"/>
    <addaction name="actionManage_Custom_Types"/>
   </widget>
   <widget class="QMenu" name="menuRecords">
    <property name="title">
     <string>Records</string>
    </property>
    <addaction name="actionNew_Record"/>
    <addaction name="actionEdit_Record"/>
    <addaction name="actionRemove_Record"/>
   </widget>
   <widget class="QMenu" name="menuExport">
    <property name="title">
     <string>Export</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuAnalyze">
    <property name="title">
     <string>Analyze</string>
    </property>
    <addaction name="actionRun_Integrity_Checks"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuRecords"/>
   <addaction name="menuFields"/>
   <addaction name="menuExport"/>
   <addaction name="menuAnalyze"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew_Project"/>
   <addaction name="actionOpen_Project"/>
   <addaction name="actionSave_Project"/>
   <addaction name="separator"/>
   <addaction name="actionNew_Record"/>
   <addaction name="actionRemove_Record"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dockWidget">
   <property name="floating">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::AllDockWidgetAreas</set>
   </property>
   <property name="windowTitle">
    <string>Error List</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QToolButton" name="toolButtonErrors">
         <property name="text">
          <string>Errors</string>
         </property>
         <property name="icon">
          <iconset>
           <normalon>:/Error</normalon>
          </iconset>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextBesideIcon</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButtonWarnings">
         <property name="text">
          <string>Warnings</string>
         </property>
         <property name="icon">
          <iconset>
           <normalon>:/Warning</normalon>
          </iconset>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextBesideIcon</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButtonMessages">
         <property name="text">
          <string>Messages</string>
         </property>
         <property name="icon">
          <iconset>
           <normalon>:/Information</normalon>
          </iconset>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextBesideIcon</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTableWidget" name="tableWidgetErrorList">
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::SingleSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About...</string>
   </property>
  </action>
  <action name="actionNew_Project">
   <property name="icon">
    <iconset>
     <normalon>:/Media/Icons/NewFile_6276.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>New Project...</string>
   </property>
  </action>
  <action name="actionSave_Project">
   <property name="icon">
    <iconset>
     <normalon>:/Media/Icons/Saveall_6518.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Save Project</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionOpen_Project">
   <property name="icon">
    <iconset>
     <normalon>:/Media/Icons/Open_6529.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Open Project...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionField_Definions">
   <property name="text">
    <string>Manage Field Definitions...</string>
   </property>
  </action>
  <action name="actionNew_Record">
   <property name="icon">
    <iconset>
     <normalon>:/Media/Icons/AddUsingNamespace_6782.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>New Record...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionEdit_Record">
   <property name="text">
    <string>Edit Record...</string>
   </property>
   <property name="shortcut">
    <string>Return</string>
   </property>
  </action>
  <action name="actionRemove_Record">
   <property name="icon">
    <iconset>
     <normalon>:/Media/Icons/RemoveUsingNamespace_10480.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Delete Record</string>
   </property>
   <property name="toolTip">
    <string>Delete Record</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actionManage_Components">
   <property name="text">
    <string>Manage Components...</string>
   </property>
  </action>
  <action name="actionManage_Custom_Types">
   <property name="text">
    <string>Manage Custom Types...</string>
   </property>
  </action>
  <action name="actionManual">
   <property name="icon">
    <iconset resource="../../../Project/tome.qrc">
     <normaloff>:/Media/Icons/StatusAnnotations_Help_and_inconclusive_32xLG_color.png</normaloff>
     <normalon>:/Media/Icons/StatusAnnotations_Help_and_inconclusive_32xSM_color.png</normalon>:/Media/Icons/StatusAnnotations_Help_and_inconclusive_32xLG_color.png</iconset>
   </property>
   <property name="text">
    <string>Manual</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionReport_a_Bug">
   <property name="text">
    <string>Report a Bug</string>
   </property>
  </action>
  <action name="actionReleases">
   <property name="text">
    <string>Releases</string>
   </property>
  </action>
  <action name="actionRun_Integrity_Checks">
   <property name="icon">
    <iconset>
     <normalon>:/Media/Icons/StackedAreaSeries_12693.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Run Integrity Checks...</string>
   </property>
   <property name="shortcut">
    <string>F6</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../../../Project/tome.qrc"/>
 </resources>
 <connections/>
</ui>
